parts/linux/cloud-init/artifacts/cse_install.sh

```diff
@@ -229,20 +230,46 @@ installAzureCNI() {
 }
 
 extractKubeBinaries() {
-    K8S_VERSION=$1
-    KUBE_BINARY_URL=$2
+    local k8s_version="$1"
```

Comment:
nit: any particular reason to lower case the vars? I dont have a strong preference but i'd like the format to be consistent with the other variables used in the func like K8S_DOWNLOADS_DIR, and the exit code.

Reply:
My understanding is that the local variables in lower case is the convention - K8S_DOWNLOADS_DIR and exit codes are global and/or exported.